docs/UserGuide.md

```diff
@@ -24,44 +24,42 @@ Get started by following the steps in this user guide! Experience the difference
 
 --------------------------------------------------------------------------------------------------------------------
 ## Introducing PoochPlanner
-PoochPlanner is your go-to address book, tailor-made for dog cafe owners like you. With four main contact types—**person, staff, supplier, maintainer**—each equipped with its own set of attributes and functions, managing your contacts has never been easier.
+PoochPlanner is your go-to address book, tailor-made for dog cafe owners like you. With four main contact types — **person, staff, supplier, maintainer** — each equipped with its own set of attributes and functions, managing your contacts has never been easier.
 
 Contact | Definition and Examples
 --------|------------------
-Person | Any persons that do not hold a special relationship.<br> E.g. Neighbours, Work Friends.
-Staff | Any full-time or part-time employees that do not do any specialised work.<br> E.g. Waiters, Waitresses, Dishwashers.
-Supplier | Any logistics partner that supplies stock for your dog cafe.<br> E.g. Kibbles Supplier, Coffee Beans Supplier.
-Maintainer | Any externally hired specialised persons who take care of the dogs.<br> E.g. Dog Groomers, Vet.
+Person | Any persons that do not hold a special relationship.<br> E.g. Neighbours, Work friends.
```

Comment:
Standardise capitalisation. e.g. Work Friends or work friends. same for Kibbles Supplier, Coffee Beans Supplier, Dog Groomers in the lines below.

My take is not to capitalise as these are not proper nouns.

Reply:
noted !! will change to remove capitalisation